scripts/build-windows/kcl-doc.go

```diff
@@ -57,29 +57,22 @@ func Install_Kclvm(installed_path string) {
 		os.Exit(1)
 	}
 
-	// Check if "installed" file exists
-	outputPath := filepath.Join(installed_path, "kclvm_installed")
-	if _, err := os.Stat(outputPath); err == nil {
+	cmd = exec.Command("cmd", "/C", "python3", "-m", "pip", "show", "kclvm")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+
+	// Check if kclvm has been installed.
+	if err := cmd.Run(); err == nil && out.String() != "WARNING: Package(s) not found: kclvm" {
```

Comment:
The judgment here is a little hard. For example, strings.Contains(out.String(), "Package(s) not found")

Reply:
fixed